bliss.js

```diff
@@ -1,6 +1,38 @@
 (function() {
 "use strict";
 
+function overload(callback, start) {
+	start = start === undefined ? 1 : start;
+
+	return function() {
+		var isSingleParam = $.type(arguments[start]) === 'object', 
```

Comment:
I would rather if we checked whether it's NOT a single param, by checking if it's a string. That way, people can still use objects whose [[Class]] is different from "Object". Not that it's common, but why not allow it if we can.